examples/google-cloud-build/cloudbuild-jib-gradle+cache.yaml

```diff
@@ -0,0 +1,69 @@
+substitutions:
+  _GCS_CACHE_BUCKET: <your bucket>
+  _IMAGE_NAME: my-image
+
+steps:
+
+# First, loads the cached files from GCS if they exist.
+- waitFor:
+  - '-'
+  name: gcr.io/cloud-builders/gsutil
+  dir: /builder/home
+  entrypoint: /bin/bash
+  args:
+  - -c
+  - |
+    mkdir /builder/home/.gradle &&
```

Comment:
Just to confirm: is everything under `/builder/home` persistent across all the steps?

Reply:
Yes